tests/test_indexing.cc

```diff
@@ -332,4 +332,10 @@ TEST(test_indexing, test_matrix_symmetric_subset) {
   EXPECT_EQ(symmetric_subset(x, idx), expected);
 }
 
+TEST(test_indexing, test_is_subsetable) {
+  EXPECT_TRUE(bool(details::is_subsetable<std::vector<int>>::value));
+  EXPECT_TRUE(bool(details::is_subsetable<RegressionDataset<int>>::value));
```

Comment:
is there a test for this one?

Reply:
I'll need more context, which "one"?

Reply:
PR lgtm, trying to understand the `is_subsetable` class, under what type will `is_subsetable` be true?

Reply:
`is_subsetable<X>` is looking to see if there's a method with the signature: `X = subset(X, std::vector<std::size_t>)`.  For example, you can subset vectors ([here](https://github.com/swift-nav/albatross/blob/master/include/albatross/src/indexing/subset.hpp#L22)) and you can subset datasets ([here](https://github.com/swift-nav/albatross/blob/master/include/albatross/src/core/dataset.hpp#L73)), but there is not way to subset a `map<int, int>`.